src/auth/screens/language-settings.js

```diff
@@ -15,4 +15,8 @@ export default [
     code: 'tr',
     name: 'Türkçe',
   },
+  {
+    code: 'ptBr',
```

Comment:
The code must be written with a hyphen (`pr-br`); It will then be used in the library `moment`, and there it is necessary to write the language through a hyphen.

Reply:
Sorry, but I'm not sure how to proceed... Shouldn't this be the same name I use on the file and to export the translation (ptBr)? Because when I try to use `pt-br`, selecting the language doesn't work and it fallbacks to English.

Reply:
@lex111, sorry, but I'm not sure how to proceed... Shouldn't this be the same name I use on the file and to export the translation (ptBr)? Because when I try to use pt-br, selecting the language doesn't work and it fallbacks to English.